MathSoc/mathsoc-website
# Set up Google Authentication

Resolves #167.

This hides all of the admin _pages_ behind Google authentication.

It does **not** hide write endpoints yet.  That shouldn't take long to enable though once this is in.

How to test:
* Go to staging9000.mathsoc.uwaterloo.ca while on eduroam or VPN
* Try to access `/admin` <3
* Try to access `/exam-bank`.  You should be able to successfully see the exam bank page (it's currently empty and has no exams), but still should not be able to access `/admin`.

(massive thanks to @ad-world for getting so much of the authentication flow to a point where it was easy to add on a new auth strategy)
